Dockerfile.dapper

```diff
@@ -1,25 +1,20 @@
-FROM golang:1.16.9-buster
+FROM registry.suse.com/bci/golang:1.17
 
-ENV ARCH amd64
+ARG DAPPER_HOST_ARCH="amd64"
+ENV ARCH=${DAPPER_HOST_ARCH}
 
 ARG http_proxy=$http_proxy
 ARG https_proxy=$https_proxy
 ARG no_proxy=$no_proxy
 
-RUN apt-get update && \
-    apt-get install -y apt-transport-https ca-certificates
+RUN zypper clean -a
```

Comment:
`RUN zypper -n clean -a` needs to be after the `zypper install`, so it can properly clear cache files after the install.

Reply:
Ahh, I see having it before was removing the warning below. This could be due to the base image not running `zypper clean`. Would it be worth running before and after? Also, do we need the `-n` on the clean command?

```
Refreshing service 'container-suseconnect-zypp'.
Warning: Skipping service 'container-suseconnect-zypp' because of the above error.
Problem retrieving the repository index file for service 'container-suseconnect-zypp':
[container-suseconnect-zypp|file:/usr/lib/zypp/plugins/services/container-suseconnect-zypp] 
```